cypress/integration/focus-trap-demo.spec.js

```diff
@@ -55,4 +54,73 @@ describe('<FocusTrap> component', () => {
     cy.focused().should(([nextFocusedEl]) =>
       expect(nextFocusedEl).not.equal(previousFocusedEl));
   });
+
+  it('On trap mounts and activates, focus on manually specified input element', () => {
+    cy.get('#demo-ffne')
+      .as('testRoot');
+
+    // activate trap
+    cy.get('@testRoot').findByRole('button', { name: 'activate trap' })
+      .click();
+
+    // instead of next tab-order element being focused, element specified should be focused
+    cy.get('@testRoot').findByRole('textbox', { name: 'Initially focused input' }).should('be.focused');
+  });
+
+  it('Escape key does not deactivate trap. Instead, click on "deactivate trap" to deactivate trap', () => {
+    cy.get('#demo-ffne')
+      .as('testRoot');
+
+    // activate trap
+    cy.get('@testRoot').findByRole('button', { name: 'activate trap' }).as('lastlyFocusedElementBeforeTrapIsActivated')
+      .click();
+
+    // trying deactivate trap by ESC
+    cy.get('@testRoot').findByRole('textbox', { name: 'Initially focused input' }).as('trapChild').focus().type('{esc}');
+
+    // ESC does not deactivate the trap
+    cy.get('@trapChild').should('exist').should('be.focused');
+
+    // click on deactivate trap button to deactivate trap
+    cy.get('@testRoot').findByRole('button', { name: 'deactivate trap' }).click();
+    cy.get('@trapChild').should('not.exist');
+    cy.get('@lastlyFocusedElementBeforeTrapIsActivated').should('be.focused');
+  });
+
+  it('Can be deactivated while staying mounted  ', () => {
+    cy.get('#demo-special-element')
+      .as('testRoot');
+
+    // activate trap
+    cy.get('@testRoot').findByRole('button', { name: 'activate trap' }).as('lastlyFocusedElementBeforeTrapIsActivated')
+      .click();
+
+    // stay mounted after deactivation
+    cy.get('@testRoot').findByRole('button', { name: 'deactivate trap' }).as('trapChild').click();
+    cy.get('@lastlyFocusedElementBeforeTrapIsActivated').should('be.focused');
+    cy.get('@trapChild').should('exist');
+  });
+
+  it('Can click outside of trap to deactivate and click carries through  ', () => {
+    cy.get('#demo-special-element')
+      .as('testRoot');
+
+    // activate trap
+    cy.get('@testRoot').findByRole('button', { name: 'activate trap' }).click();
+
+    // click outside to deactivate trap and also click carries through
+    cy.findByRole('button', { name: 'pass thru click' }).click().should('be.focused');
```

Comment:
I was trying to just verify `document.activeElement` when click outside, but `Chrome` and `Firefox` has different behavior on clicking outside.

`Chrome` focuses on lastly focused element while `Firefox` focusing on `<body>`. Also it's not clear if it's just trap losing focus(deactivate) and also click passes thru. 

So I update the demo page a little at [new button](https://github.com/focus-trap/focus-trap-react/pull/103/files#diff-a4c3e7d6c4e1076bff9fb47c75a32f9cR43) so it clearly shows it's not only focus is freed but also click passes thru.